src/main/java/com/hubspot/jinjava/lib/filter/SliceFilter.java

```diff
@@ -79,21 +82,41 @@ public Object filter(Object var, JinjavaInterpreter interpreter, String... args)
         0,
         args[0]
       );
+    } else if (slices > MAX_SLICES) {
+      interpreter.addError(
+        new TemplateError(
+          TemplateError.ErrorType.WARNING,
+          TemplateError.ErrorReason.OVER_LIMIT,
+          TemplateError.ErrorItem.FILTER,
+          "The limit input value is too large, it's been reduced to " + MAX_SLICES,
```

Comment:
Can you refer to the `slices` count as described in the docs on line 26?
Also please add the 1000 limit to those docs.

Reply:
added